Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -662,11 +664,11 @@ private func resolveModuleAliases(with packageBuilders: [ResolvedPackageBuilder]
     }
 
     // Track targets that need module aliases for each package
-    for packageBuilder in packageBuilders {
-        for prod in packageBuilder.package.products {
-            var list = prod.targets.map{$0.dependencies}.flatMap{$0}.compactMap{$0.target?.name}
-            list.append(contentsOf: prod.targets.map{$0.name})
-            aliasTracker.addAliasesForTargets(list, for: prod.name, in: packageBuilder.package.identity)
+    packageBuilders.forEach { packageBuilder in
+        packageBuilder.package.products.forEach { produdct in
```

Comment:
@abertelrud told me in the past that they preferred for ... in in SwiftPM code base

Reply:
ok will revert back to `for...`